apps/src/aichat/views/model-customization-workspace.module.scss

```diff
@@ -0,0 +1,48 @@
+@import 'color';
+@import '../../mixins.scss';
+
+%full-width {
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.modelCustomizationWorkspace {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+
+  textarea {
```

Comment:
nit: personal preference, but should we use class names here instead of selecting by element type?

Reply:
haha I thought I was being slick by not having to use class names :) Do you just think it's easier if you see the class name inline when you're looking at some component, vs. in this case you're seeing some styling applied without having an explicit class name applied?

Reply:
Haha I usually just opt for classnames to have more control over what elements should receive which styles - just in case we add another `textarea` here but want to apply different styles, for example. To your question I personally like to see the classname inline since it's also easier for tracking down where a particular set of styles originate from in the CSS. That being said, I could go either way with this specific example because we'd probably want all `textarea`s to look the same in this component anyway.

Reply:
Ok cool -- I'm going to merge so I can untangle merge conflicts with this branch / your levelbuilder branch in a follow-up I have going to load in levelbuilder config. Happy to update this down the line though!